.github/workflows/hw-event-proxy.yaml

```diff
@@ -24,6 +24,7 @@ jobs:
           skip-build-cache: true
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
           version: v1.41
+          working-directory: ./hw-event-proxy
   unit-tests:
     name: Unit Tests
     runs-on: ubuntu-20.04
```

Comment:
Ubuntu?

Reply:
seems to be common among github projects